packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissibleInput.tsx

```diff
@@ -81,14 +84,35 @@ export function useInternalChipDismissibleInput({
 
     handleBlur: () => {
       if (shouldCancelBlur) return;
+
+      if (
+        submitInputOnFocusShift &&
+        searchValue.length > 0 &&
+        allOptions.length > 0
+      ) {
+        // If there's a custom option, select it. Otherwise select the best match
+        const optionToSelect = canAddCustomOption
+          ? allOptions[allOptions.length - 1]
+          : allOptions[0];
```

Comment:
what's the relevance of `allOptions.length - 1`?

is the last option when you can custom add the "add _____ as whatever"?